dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/agent/FamilyNameController.java

```diff
@@ -62,17 +63,25 @@ public PageResponse<FamilyName> find(
 
   @Operation(summary = "Get a familyname by namespace and id")
   @GetMapping(
-      value = {
-        "/v5/familynames/identifier/{namespace}:{id}",
-        "/v5/familynames/identifier/{namespace}:{id}.json"
-      },
+      value = {"/v5/familynames/identifier/**"},
```

Comment:
Könnte man hier noch einen Kommentar ergänzen, der die Verwendung des Endpunkts zeigt? Damit man nicht erst den Code lesen muss...

Reply:
Wird in eigenen PR ausgelagert